package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vaadin/bundles",
-  "version": "24.3.0-alpha4",
+  "version": "24.3.0-alpha5",
```

Comment:
This needs to be changed back to `alpha4`.

Reply:
okay

Reply:
updated.

Reply:
oops, test failed.. seems this version should be the same with the component version